src/Business Foundation/App/NoSeriesCopilot/src/Copilot/NoSeriesProposal.Table.al

```diff
@@ -0,0 +1,46 @@
+table 391 "No. Series Proposal"
+{
+    TableType = Temporary;
+
```

Comment:
DataClasification missing

Reply:
This is a temporary table. No classification needed, as data isn't stored.